CMakeLists.txt

```diff
@@ -163,3 +163,18 @@ include (package)
 # ===========================================================================
 
 include (SeqAnCtdSetup)
+
+# ===========================================================================
+# Install all C++ header files
+# ===========================================================================
+
+install (DIRECTORY include/seqan DESTINATION "${CMAKE_INSTALL_PREFIX}/include")
+
```

Comment:
Can you please revert the changes above and only keep the additions for the pkg-config.
We have special build flags for packaging. See here: http://seqan.readthedocs.org/en/master/BuildManual/UsingTheSeqAnBuildSystem.html#packaging-library-releases